app/grandchallenge/evaluation/admin.py

```diff
@@ -58,6 +65,84 @@ def clean(self):
         return cleaned_data
 
 
+class ConfigureAlgorithmPhasesForm(Form):
+    phases = ModelMultipleChoiceField(
+        queryset=Phase.objects.select_related("challenge")
+        .filter(submission_kind=SubmissionKindChoices.CSV)
+        .all()
+    )
+    algorithm_inputs = ModelMultipleChoiceField(
+        queryset=ComponentInterface.objects.all()
+    )
+    algorithm_outputs = ModelMultipleChoiceField(
+        queryset=ComponentInterface.objects.all()
+    )
+
+    def clean(self):
+        cleaned_data = super().clean()
+        try:
+            for phase in self.cleaned_data["phases"]:
```

Comment:
Does this matter? I thought that the slug field will automatically choose a free slug?

Reply:
Ah you might be right, I will double check.